CodeiumVS/Commands.cs

```diff
@@ -91,8 +91,19 @@ protected override void BeforeQueryStatus(EventArgs e)
         is_visible = Command.Visible = ThreadHelper.JoinableTaskFactory.Run(async delegate
         {
             is_function = false;
-            docView = await VS.Documents.GetActiveDocumentViewAsync();
-            if (docView?.TextView == null) return false;
+
+            // any interactions with the `IVsTextView` should be done on the main thread
+            await ThreadHelper.JoinableTaskFactory.SwitchToMainThreadAsync();
```

Comment:
It only throws an exception when right-clicking the diff view. I wonder why it doesn't execute on the main thread, only for the diff view?